pkg/apis/cert/v1alpha1/issuer.go

```diff
@@ -69,6 +69,31 @@ type ACMESpec struct {
 	// PrivateKeySecretRef is the secret ref to the ACME private key.
 	// +optional
 	PrivateKeySecretRef *corev1.SecretReference `json:"privateKeySecretRef,omitempty"`
+
+	// ACMEExternalAccountBinding is a reference to a CA external account of the ACME server.
+	// +optional
+	ExternalAccountBinding *ACMEExternalAccountBinding `json:"externalAccountBinding,omitempty"`
+
+	// NoDNSEntryCreation is a flag to disable creating of DNS entries/records for a DNS Challenge.
+	// For some setups, e.g. when using ExternalAccountBinding with a SAP DigiCert account, the DNS challenges are only performed pro forma.
+	// +optional
+	NoDNSEntryCreation *bool `json:"noDNSEntryCreation,omitempty"`
```

Comment:
```suggestion
	SkipDNSChallenge *bool `json:"noDNSEntryCreation,omitempty"`
```

Reply:
Good point. Renamed to SkipDNSChallengeValidation, as the DNS challenge is still performed on the protocol level. But the validation is skipped from the ACME server.